shared/unattended/RHEL-6-perf-series.ks

```diff
@@ -0,0 +1,72 @@
+install
+KVM_TEST_MEDIUM
+text
+poweroff
+lang en_US.UTF-8
+keyboard us
+key --skip
+network --bootproto dhcp
+rootpw 123456
+firewall --enabled --ssh
+selinux --enforcing
+timezone --utc Asia/Shanghai
```

Comment:
@luckyh I don't think so.  The same way other kickstart files have been using other TZs, and it looks like it hasn't been a problem to Chinese user (or has it?).

If you do know of problems, or have a use case for this dinamically, please let me know.